packages/orbit-components/src/Textarea/index.tsx

```diff
@@ -186,25 +81,43 @@ const Textarea = React.forwardRef<HTMLTextAreaElement, Props>((props, ref) => {
         </FormLabel>
       )}
 
-      <StyledTextArea
+      <textarea
+        className={cx(
+          "w-full appearance-none",
+          "font-base",
+          "rounded-normal relative box-border block overflow-auto",
+          "min-h-form-box-normal",
+          "duration-fast transition-shadow ease-in-out",
+          "border border-transparent",
+          // TODO: remove when will be migrated from tmp-folder
```

Comment:
@mainframev Could you please explain this? Do we need to do something?

Reply:
our inputs like InputField, and InputFile -> no longer provide size prop and it was removed, but looks like it was missed for Textarea. In Figma, there is only one size for this component (normal). So it should be removed imo. I didn't want to do that under the chore commit in the temp folder. I asked designers and they said, that "in Figma we have one size, but it should support resize" (which is supported via resize prop)

Reply:
@mvidalgarcia I see that you kept this. No problem, can be changed any time, it's supposed to breaking, so just change the comment to something like: "TODO: remove size small for Textarea"

Reply:
Just pushed a commit addressing it